solutions/usaco-688.mdx

```diff
@@ -0,0 +1,58 @@
+---
+id: usaco-688
+source: USACO Bronze 2017 January
+title: Hoof, Paper, Scissors
+author: Ryan Chou
+---
+
+## Solution 1
+[Official Analysis](http://www.usaco.org/current/data/sol_hps_bronze_jan17.html)
+
+## Solution 2 - Generating All Permutations
+
+We can map the games to every possible permutation of the moves (hoof, paper, scissors) and find the maximum of wins. This works because $1 \leq N \leq 100$
+
+### Implementation
```

Comment:
add period after "This works because $1 \leq N \leq 100$"

add time complexity below ### Implementation

Reply:
👍

Reply:
added it :)